app/models/saved_claim.rb

```diff
@@ -148,13 +150,43 @@ def va_notification?(email_template_id)
   private
 
   def validate_schema(schema)
```

Comment:
Can you please add some docblocks or comments to these functions?
e.g. 
/**
 * Adds two numbers together.
 *
 * @param int $a The first number.
 * @param int $b The second number.
 * @return int The sum of the two numbers.
 */